konfera/tests/test_models.py

```diff
@@ -409,3 +409,15 @@ def test_status(self):
         tt.date_from = now - datetime.timedelta(days=3)
         tt.date_to = now - datetime.timedelta(days=1)
         self.assertEquals(tt.status, TicketType.STATUSES[TicketType.EXPIRED])
+
+
+class OrganizerTest(TestCase):
+
+    def setUp(self):
+        self.first_organizer = models.Organizer(title='Mysterious Organizer', street='1 Up street', city='Big City',
+                                                company_id='123',
+                                                about_us="World famous yet unknown conference organizer.")
+
+    def test_save(self):
+        self.assertTrue(self.first_organizer.save)
```

Comment:
hmm... `.save` is a method so you have to invoke it, otherwise the assertion doesn't make any sense see:

```
>>> def f(): return False
>>> bool(f)
True
```

If you called `.save()` you would see it throws an error because `street2` is a mandatory field.